allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/newDrivers/INewWebDriver.java

```diff
@@ -248,6 +250,15 @@ MenuElement elementMenu(final By selector,
 					final By subMenuSelector,
 					final By childsSubMenuSelector);
 	
+	/**
+	 * Operations on Slide
+	 */
+	HorizontalSliderElement elementHorizontalSlider(final By selector);
```

Comment:
1. First parameter name needs to be more meaningful.

Reply:
Done.